src/IECore/IFFFile.cpp

```diff
@@ -61,23 +61,23 @@ bool IFFFile::open()
 		{
 			return false;
 		}
-
```

Comment:
This file has had its formatting munged somehow.

Reply:
Yeah I hate MSVC's liberties of formatting like this, I'll need to muck around in the options to fix...

Reply:
Just noting that this is still full of unnecessary whitespace changes that will need sorting before we merge. I tend to use `git rebase --whitespace=fix master` to fix this sort of thing - maybe give that a go?

Reply:
It looks like he has fixed the formatting now, but the commit history is pretty confusing, with bad commits followed by reverts followed by good commits.  If he squashes the reverts in, then we should get a legible commit history.